scalardl-node-client-sdk.js

```diff
@@ -72,7 +74,14 @@ class ClientServiceWithBinary extends ClientServiceBase {
    * @param {Object} properties
    */
   constructor(properties) {
-    super(_createGrpcServices(properties), protobuf, properties);
+    super(
+        {
+          ..._createGrpcServices(properties),
```

Comment:
I'm just not sure about `...`. Sorry for my inexperience in js ...

Also I feel `_createGrpcServices` can be unified with the corresponding code in the web sdk ?

Reply:
> I'm just not sure about .... Sorry for my inexperience in js ...

This is `spread operator` in JavaScript.
It is a sugar syntax for many usages.

One of the usage is to merge two objects into a new object.
```
let foo = {property1: "foo"};
let bar = {property2: "bar"};
let merged = {...foo, ...bar};
console.loe(merged);
> {property1: "foo", property2: "bar"}
``` 

In our codes, we merge two grpc services
```
{
    ledgerClient: ...
    ledgerPrivileged: ...
}
```
and the signer factory
```
{
    signerFactory: ...
}
```
into one object
```
{
    ledgerClient: ...
    ledgerPrivileged: ...
    signerFactory: ...
}
```
and then pass it to ClientServiceBase.


> Also I feel _createGrpcServices can be unified with the corresponding code in the web sdk ?

Web SDK has a bit different workflow to create gRPC services. For example, we don't need to setup CA root certificates for Web SDK because it will be based on the browser's CA root bundle.

For your reference :)

Reply:
Got it ! And thank you for explaining about `...` ! (I need to learn Javascript !